core/sys/cc-gcc.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2015, Scanimetrics - http://www.scanimetrics.com
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in the
+ *    documentation and/or other materials provided with the distribution.
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived
+ *    from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE
+ * COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+/*---------------------------------------------------------------------------*/
+#ifndef _CC_GCC_H_
+#define _CC_GCC_H_
+#ifdef __GNUC__
+
+#ifndef CC_CONF_INLINE
+/* use __inline__ in case "inline" is not available for any reason */
+#define CC_CONF_INLINE __inline__
+#endif
+
+#define CC_CONF_ALIGN(n) __attribute__((__aligned__(n)))
+
+#endif //__GNUC__
+#endif //_CC_GCC_H_
```

Comment:
Right, and GitHub says that a rebase is needed too now.

It would be great if the merging rules were relaxed for such things. IMO, these are typical trivial fixes that should be part of a manual merge commit or of some fixup PR edit or of a new PR made by a maintainer because they are not worth bothering the original PR author, especially after several months of inactivity. What do you think?

Also, we are still waiting for an upvote from the core maintainers for this PR. There have been nearly 3 months of inactivity on this PR, so we should merge this PR in one month according to the new PR policy (after fixup of the above). We should probably not reset the PR counter to 2 months just for the little change requested here.